update-ca-certificates

```diff
@@ -76,6 +76,13 @@ esac
 rm -f /etc/pki/trust/.updated
 
 mkdir -p "$statedir"
+# serialize calls if we can
+if [ -z "$_CA_CERTIFICATES_LOCKED" -a -x /usr/bin/flock ]; then
```

Comment:
`-a` is deprecated

```suggestion
if [ -z "$_CA_CERTIFICATES_LOCKED" ] && [ -x /usr/bin/flock ]; then
```

Reply:
fixing shellcheck complaints would be other commit :)